src/cljs/reagent_toolbox_docs/components/radio_buttons.cljs

```diff
@@ -0,0 +1,188 @@
+;;;; Copyright © 2017 Flexpoint Tech Ltd
+
+(ns reagent-toolbox-docs.components.radio-buttons
+  (:require [reagent-toolbox-docs.layout :as layout]
+            [reagent-toolbox-docs.ui :as ui]))
+
+(defn view
+  []
+  (fn []
+    [:article
+     [:h1 "Radio Buttons"]
+
+     [:p
+      [:a {:href "https://material.google.com/components/selection-controls.html#selection-controls-radio-button"} "Radio buttons"]
+      " allow the user to select one option from a set. Use radio buttons for exclusive "
+      "selection if you think that the user needs to see all available options side-by-side. "
+      "Otherwise, consider a dropdown, which uses less space than displaying all options. They "
+      "should always be used along with RadioGroup."]
+
+     [:p "You can provide the theme for this component using the key ToolboxButton."]
+
+     [ui/display-and-eval-code
+      "(defn radio-buttons-test
+         []
```

Comment:
Please, for the examples, keep the arguments in the same line as the name to have less boilerplate lines.